site/src/docs/patterns/cookies/common-helsinki-cookies.mdx

```diff
@@ -1,31 +1,30 @@
 ---
 slug: '/patterns/cookies/common-helsinki-cookies'
-title: 'Common Helsinki cookies'
-navTitle: 'Common Helsinki cookies'
+title: 'Commonly known Helsinki cookies'
+navTitle: 'Commonly known Helsinki cookies'
 ---
 
 import LeadParagraph from '../../../components/LeadParagraph';
 import ExternalLink from '../../../components/ExternalLink';
 import StatusLabel from '../../../components/StatusLabel';
 
-# Common Helsinki cookies
+# Commonly known Helsinki cookies
 
 <StatusLabel type="error">Draft</StatusLabel>
 
 <LeadParagraph>
-  Using common Helsinki cookies brings the benefit of consistent content across services that use the same components.
+  Using commonly Helsinki cookies brings the benefit of consistent content across services that use the same components.
 </LeadParagraph>
 
 ## Key principles of using common cookies
 
-- You can use common Helsinki cookies if your service is using a \*.hel.fi domain.
-- When common cookies are available, use them instead of creating custom cookies.
-- When a common cookie is used, it must be used **exactly** as described on this page. If you modify the cookie in any way, it is no longer a common cookie.
-- If your service is using custom cookies alongside common cookies, you still need to ask for consent for the custom cookies (if they are considered optional).
-- You can either use all or a part of the common cookies.
-- Note that some of the services (e.g., Matomo) may require multiple common cookies for it to work.
+- You can use commonly known Helsinki cookies if your service is using a \*.hel.fi domain.
+- When known cookies are available, use them instead of creating custom cookies. This way they are at least consistent even they are handled on service level.
+- If your service is using cookies, you need to ask for consent for the cookies (if they are considered optional).
+- You can either use all or a part of the commonly cookies.
+- Note that some of the services (e.g., Matomo) may require multiple cookies for it to work.
 
-## Common cookies between \*.hel.fi sites
+## Commonly known cookies for \*.hel.fi sites
 
 The following is a list of approved common cookies for common services like Matomo or Tunnistamo. You can find this list as a JSON object in the <ExternalLink href="https://github.com/City-of-Helsinki/helsinki-design-system/blob/master/packages/react/src/components/cookieConsent/getContent.ts" openInNewTab openInNewTabAriaLabel="Opens in a new tab" openInExternalDomainAriaLabel="Opens a different website">Helsinki Design System GitHub repository</ExternalLink>.
```

Comment:
This is false, all services have individual cookies. Across hel.fi domains the descriptions of some cookies can be shared, but the cookie settings are per site and never set to *.hel.fi domain.

Reply:
So @Arkkimaagi  is the whole page irrelevant? (common-helsinki-cookies) or what should be left here?